res/css/views/rooms/_EventTile.scss

```diff
@@ -20,6 +20,10 @@ $left-gutter: 64px;
 .mx_EventTile {
     .mx_EventTile_receiptSent,
     .mx_EventTile_receiptSending {
+        // Give it some dimensions so the tooltip can position properly
+        display: inline-block;
+        width: 14px;
+        height: 14px;
```

Comment:
In the before, it was just positioned by the a random ~50px offset from the right side which was fixed for all `Alignment.Left` tooltips regardless of the element size. Now it positions from the actual element.

Before | After
--- | ---
<img width="291" alt="" src="https://user-images.githubusercontent.com/558581/149612687-eecdf36f-8c04-4162-b1d1-82c9a995277c.png"> | <img width="301" alt="" src="https://user-images.githubusercontent.com/558581/149612697-1597caa9-4fd6-4861-a70d-0bcc7dc5a245.png">

Reply:
The v-align is better here but still slightly off, now the tooltip looks a couple px too high

Reply:
I can change the vertical alignment on this specific tooltip but this probably comes down to the vertical tooltip calculation code which I didn't touch in this PR and also has arbitrary offsets in it. Ideally that code should vertically center things just fine. We can address the vertical problems in another PR because I don't want to keep re-doing the before/after screenshots, etc

It's 20px above and below the read receipt now:

<img width="291" alt="" src="https://user-images.githubusercontent.com/558581/149829602-d5664a35-6fdd-48ce-9882-5dcf0c4d5bdf.png">